pkg/proto/pbgo/core/autodiscovery.pb.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.35.2
+// 	protoc-gen-go v1.36.1
```

Comment:
this change should not be part of this PR. 

it seems that we still have an issue with the CI validation to verify that the invoke generate protobuf command runs when the tooling is updated.